test/exercises/acronym_test.rb

```diff
@@ -0,0 +1,93 @@
+require "test_helper"
+require 'pry'
```

Comment:
This shouldn't be here unless it is actively used in this test file.  I believe it is left here after some debugging was happening. 

```suggestion
```

Reply:
Good catch!